styleguide/source/assets/js/jump-nav.js

```diff
@@ -9,4 +10,44 @@ $(document).ready(function() {
     $(this).parents('#jump-nav').toggleClass('is-closed');
   });
 
-});
+  // sticky jumpnav
+  var StickyElement = function(node){
+    var doc = $(document),
+        fixed = false,
+        anchor = node.find('.jumpnav-anchor'),
+        body = doc.find('.article-body'),
+        content = node.find('.jumpnav-content');
+
+    var onScroll = function(e){
+      var docTop = doc.scrollTop(),
+          anchorTop = anchor.offset().top,
+          stickyBottom = content.offset().top + content.height(),
+          bodyBottom = body.offset().top + body.height();
+
+      if(docTop > anchorTop){
+        if(!fixed){
+          anchor.height(content.outerHeight());
+          content.addClass('fixed');
+          fixed = true;
+        }
+      }  else   {
+        if(fixed){
+          anchor.height(0);
+          content.removeClass('fixed');
+          fixed = false;
+        }
+      }
+
+      if(stickyBottom > bodyBottom){
+        content.hide();
+      } else {
+        content.fadeIn(250);
+      }
+    };
+
+    $(window).on('scroll', onScroll);
+  };
+
+var demo = new StickyElement($('#sticky-jumpnav'));
```

Comment:
@martensc ... I'm not sure. I think it doesn't work if you don't have it. but we could change the var to something else if demo doesn't make sense